test/spec.js

```diff
@@ -280,4 +281,35 @@ describe('L.Sync', function () {
             a._syncMaps.should.eql([]);
         });
     });
+    
+    describe('sync with circle marker', function () {
```

Comment:
It should be easy to figure out how to write trivial mocha tests, this however, might require some extra steps, since we need to simulate a mouse event over a map and check if the location of the marker on the synced map equals the location of the mouse.

I think a single test testing for regression with the `syncCursor` option enebled makes sense, if it has relevant descriptions.